internal/engine/compiler/engine.go

```diff
@@ -785,7 +785,11 @@ func (ce *callEngine) deferredOnCall(recovered interface{}) (err error) {
 					sources = p.sourceModule.DWARFLines.Line(offset)
 				}
 			}
-			builder.AddFrame(def.DebugName(), def.ParamTypes(), def.ResultTypes(), sources)
+			name := def.DebugName()
+			if expf, ok := def.(experimental.FunctionDefinition); ok {
+				name = expf.HumanName()
+			}
```

Comment:
Is it worth putting a function to do this in the `experimental` package so we don't repeat it in the interpreter and compiler?

```go
func DebugName(def api.FunctionDefinition) string {
   ...
}
```

Reply:
Would it be confusing for the users of `experimental` that they have access to that information both using a function and through the `experimental.FunctionDefinition` API?

Reply:
Maybe this is the good place for `FunctionHumanReadableName` or `BestDebugName` (doesn't have to be called the same as the method name).

Reply:
This seems unnecessary now as `DebugName()` is repurposed.